### ISSUE TITLE
Scheduler don't run (0X1)

### ISSUE THREAD

**Issue by waxweazle2001** [Smurf-IV/Elucidate#48]
I got the newest Version of the Snapraid GUI, I set a schedule to write the data to the parity but i noticed as i sync the data the day after that the schedule don't run.
Now i saw that the schedule give the last run result of (0x1), so i asked google and they said that I should set up "run with highest privileges", I want to do this and have it optimized for win 10 pro but when i gave my password in the box then the elucidate give me an failure and cant set the option. Any Idea??

It's not so good if I can't be sure that the server make the sync in the night when i set it, if any hdd make a fault i don't can restore the data. So it would be nice if it runs, although i don't find the schedule in the windows schedules as it was in earlier versions is this right?

Greetings
Rene

**Comment by Smurf-IV** [comment]
The Schedule functionality is provided by a nuget package, so cannot comment, on it too much.
What was the failure ?
I'm not sure what you mean by you last question?

**Comment by waxweazle2001** [comment]
I mean that in the old Version the Schedule was also visible in the Windows Schedule Plan now there is no Schedule if I make one in the Snapraid GUI.
The failure was (0x1), here is the Screenshot: [https://www.directupload.net/file/d/5405/94ifr57a_jpg.htm](url) and as I read about the failure you should check the "run with highest privilege" box. But if i do so, and after I wrote in my Password the GUI make the following failure: [https://www.directupload.net/file/d/5405/aeltnnxz_jpg.htm](url).

Hope this helps.
Greetings
Rene

**Comment by Smurf-IV** [comment]
The picture links do not work, Edit the above and just past the images in, then github will store them for everyone to see.

**Comment by waxweazle2001** [comment]
I mean that in the old Version the Schedule was also visible in the Windows Schedule Plan now there is no Schedule if I make one in the Snapraid GUI.
The failure was (0x1), here is the Screenshot: 

![94ifr57a](https://user-images.githubusercontent.com/22542230/55244438-7cb37b80-5241-11e9-9990-b2e1d9a1b1d1.jpg)

and as I read about the failure you should check the "run with highest privilege" box. But if i do so, and after I wrote in my Password the GUI make the following failure: 

![aeltnnxz](https://user-images.githubusercontent.com/22542230/55244472-8ccb5b00-5241-11e9-957f-e60878c4d556.jpg)

Hope this helps.
Greetings
Rene

**Comment by tdwilli1** [comment]
I'm having the same issue. After a sync schedule is setup it won't run. it just reports as completed with 0x1.
The manual sync runs fine. There must be something wrong in the command line. This is what gets put in the arguments line:
-File "C:\Program Files\Elucidate\Elucidate GUI\snapraid.ps1" -exe "C:\Snapraid\snapraid.exe" -args "--log \"C:\ProgramData\Elucidate\logs\<DATETIME> sync.log\"  -c \"C:\Snapraid\snapraid.conf\" sync"

No matter if I try to run this from your GUI or from task manager it won't execute a sync.

**Comment by tdwilli1** [comment]
Just installed the new release and if fixed my issue. Thanks.

**Comment by hyyz17200** [comment]
I still have this problem even in newest version.
This is the screenshot.
![QQ截图20190713234106](https://user-images.githubusercontent.com/4502808/61173573-b8433d00-a5c7-11e9-854d-3a52842d1476.png)

**Comment by Smurf-IV** [comment]
@BlueBlock ?

**Comment by BlueBlock** [comment]
I'll see if I can recreate the issue.

**Comment by BlueBlock** [comment]
I reproduced it only 1 time and can't again. The code looks correct so I'm thinking the scheduler nuget package might have/had a bug. I'll commit an update to the packages.

**Comment by Smurf-IV** [comment]
https://github.com/Smurf-IV/Elucidate/releases/tag/V19.8.993.4
Update NuGet Packages and re-issue installer

    CommandLine 2.50 -> 2.60
    KryptonToolkitSuite5472 5.472.1222 -> 5.472.1251
    NLog 4.6.5 -> 4.6.5
    ByteSize 2.0.0-alpha2 -> 2.0.0-beta1
    TaskScheduler 2.8.11 -> 2.8.15
    TaskSchedulerEditor 2.8.11 -> 2.8.15

**Comment by hyyz17200** [comment]
I updated. No any improvement.

**Comment by jakebullet70** [comment]
A note: I use Win 2008 Server. Scheduler DOES NOT run. 
My son uses Win 2016 server his DOES run.
OS issues?

**Comment by jakebullet70** [comment]
OK, here is what I did to get them to run in Server 2008. Seems like the snapraid.exe is blocked.
So in properties I unblocked it. It is running now on the scheduler. 

![unblock1](https://user-images.githubusercontent.com/14124210/66018367-671c5880-e4e7-11e9-9740-9238cd3b2c4e.png)

**Comment by hyyz17200** [comment]
I am using Windows10 Pro and has no block. Still cannot run in the scheduler.

**Comment by jakebullet70** [comment]
Run it directly from the scheduler. That is where I found a popup that stopped me from running. See if you are getting a popup.

**Comment by jakebullet70** [comment]
Another issue here is if building from source the powershell script is not sign and therefor will fail to run in the scheduler.

**Comment by Smurf-IV** [comment]
@BlueBlock Do you think that this script should be removed and replaced with just calling Elucidate with the commandLine actions ?

**Comment by Smurf-IV** [comment]
#64
